src/js/bs4/settings.js

```diff
@@ -0,0 +1,339 @@
+define([
+  'summernote/bs4/ui',
+  'summernote/base/module/Editor',
+  'summernote/base/module/Clipboard',
+  'summernote/base/module/Dropzone',
+  'summernote/base/module/Codeview',
+  'summernote/base/module/Statusbar',
+  'summernote/base/module/Fullscreen',
+  'summernote/base/module/Handle',
+  'summernote/base/module/AutoLink',
+  'summernote/bs3/module/Button',
+  'summernote/bs3/module/Toolbar',
+  'summernote/bs3/module/LinkDialog',
+  'summernote/bs3/module/LinkPopover',
+  'summernote/bs3/module/ImageDialog',
+  'summernote/bs3/module/VideoDialog',
+  'summernote/bs3/module/ImagePopover',
+  'summernote/bs3/module/HelpDialog',
+  'summernote/bs3/module/AirPopover',
+  'summernote/bs3/module/HintPopover',
+  'summernote/bs3/module/SpecialCharDialog'
+], function (
+  ui,
+  Editor, Clipboard, Dropzone, Codeview, Statusbar, Fullscreen, Handle, AutoLink,
+  Button, Toolbar, LinkDialog, LinkPopover, ImageDialog, VideoDialog, ImagePopover, HelpDialog, AirPopover, HintPopover, SpecialCharDialog
+) {
+
+  $.summernote = $.extend($.summernote, {
+    version: '@VERSION',
+    ui: ui,
+
+    lang: {
```

Comment:
Please remove lang config here. It was extracted to `summernote-en-US`.
Please refer to the following links.

https://github.com/summernote/summernote/blob/develop/src/js/bs3/settings.js#L3
https://github.com/summernote/summernote/blob/develop/src/js/base/summernote-en-US.js